linux/meta/heka.yml

```diff
@@ -43,4 +43,39 @@ log_collector:
         grace_interval: 30
         hostname: '{{ grains.host }}'
 metric_collector:
-  filter: {}
+  trigger:
+    linux_system_cpu_critical:
+      description: 'The CPU usage is too high.'
+      severity: critical
+      rules:
+      - metric: cpu_wait
+        relational_operator: '>='
+        threshold: 35
+        window: 120
+        periods: 0
+        function: avg
+      - metric: cpu_idle
+        relational_operator: <=
+        threshold: 5
+        window: 120
+        function: avg
+    linux_system_cpu_warning:
+      description: 'The CPU wait times are high.'
+      severity: critical
+      rules:
+      - metric: cpu_wait
+        relational_operator: '>='
+        threshold: 15
+        window: 120
+        periods: 0
+        function: avg
+  alarm:
+    linux_system_cpu:
+      notifications: False
+      alerting: True
+      triggers:
+      - linux_system_cpu_warning
+      - linux_system_cpu_critical
+      dimension:
+        node_role: control
+        logical_name: cpu
```

Comment:
Please remove the dimensions entirely. This alarm will be installer on all the nodes, so we cannot add a `node_role`.